### ISSUE TITLE
date param error

### ISSUE THREAD

**Issue by benbro** [erleans/pgo#31]
This query gives badarg error in v0.10.0:
```erlang
pgo:query(<<"SELECT $1::date;">>, [{2020,1,1}]).
```
It works in v0.8.0:
```erlang
#{command => select,num_rows => 1,rows => [{{2020,1,1}}]}
```

**Comment by tsloughter** [comment]
Seems I need to make a new release because when I run this locally on master it works:

```erlang
pgo:query(<<"SELECT $1::date;">>, [{2020,1,1}]).
#{command => select,num_rows => 1,rows => [{{2020,1,1}}]}
```

**Comment by tsloughter** [comment]
Published v0.11.0 which should work.

**Comment by benbro** [comment]
v0.11.0 works. thanks
